Stratis.StratisD/Program.cs

```diff
@@ -32,6 +33,7 @@ public static void Main(string[] args)
                 .UseStratisConsensus()
                 .UseBlockStore()
                 .UseMempool()
+                .UseWallet()
```

Comment:
The daemon currently is built with .AddPowPosMining() 
which has the dependency on the wallet manager. Should we remove PowPosMining then? Or have another extension method that adds the wallet dependencies that mining needs?